content/scalers/aws-sqs.md

```diff
@@ -13,48 +13,78 @@ Scale applications based on AWS SQS Queue.
 
 ### Trigger Specification
 
-This specification describes the `aws-sqs-queue` trigger for AWS SQS Queue.
+This specification describes the `aws-sqs-queue` trigger that scales based on an AWS SQS Queue.
 
 ```yaml
 triggers:
   - type: aws-sqs-queue
+    authenticationRef: 
```

Comment:
Don't think it's a must to use this here as it's not really specific to this scaler, no?